lk2nd/device/dts/msm8226/msm8926-v2-720p-mtp.dts

```diff
@@ -26,5 +26,35 @@
 			qcom,mdss_dsi_otm1283a_lide_cpt_hd_video {};
 		};
 	};
-};
 
+	sony-eagle {
+		/*
+		 * To build for sony-eagle, set the following flag in your local.mk:
```

Comment:
Can you not just add it when running `make`? `make LK2ND_DISPLAY=innolux_qhd_video ...`